pennylane/operation.py

```diff
@@ -898,6 +944,32 @@ def num_params(self):
         """
         return self._num_params
 
+    @property
+    def ndim_params(self):
+        """Number of dimensions per trainable parameter of the operator.
+
+        By default, this property returns the numbers of dimensions of the parameters used
+        for the operator creation. If the parameter sizes for an operator subclass are fixed,
+        this property can be overwritten to return the fixed value.
+
+        Returns:
+            tuple: Number of dimensions for each trainable parameter.
+        """
+        return self._ndim_params
+
+    @property
+    def batch_size(self):
+        """Number of dimensions per trainable parameter of the operator.
+
+        By default, this property returns the numbers of dimensions of the parameters used
+        for the operator creation. If the parameter sizes for an operator subclass are fixed,
+        this property can be overwritten to return the fixed value.
+
+        Returns:
+            tuple: Number of dimensions for each trainable parameter.
```

Comment:
Looks like this docstring is a copy of the one above?

Reply:
It is :facepalm: Thanks :)